components/plugin_marketplace/marketplace_item/marketplace_item.js

```diff
@@ -0,0 +1,426 @@
+                />
+            );
+        }
+
+        let button = (
+            <button
+                onClick={this.onInstall}
+                className='btn btn-primary'
+                disabled={this.props.installing || this.props.downloadUrl === ''}
+            >
+                <LoadingWrapper
+                    loading={this.props.installing}
+                    text={localizeMessage('marketplace_modal.installing', 'Installing...')}
+                >
+                    {actionButton}
+                </LoadingWrapper>
+
+            </button>
+        );
+
+        if (this.props.installedVersion !== '' && !this.props.installing && !this.props.error) {
+            button = (
+                <Link
+                    to={'/admin_console/plugins/plugin_' + this.props.id}
+                >
+                    <button
+                        onClick={this.onConfigure}
+                        className='btn btn-outline'
+                    >
+                        <FormattedMessage
+                            id='marketplace_modal.list.configure'
+                            defaultMessage='Configure'
+                        />
+                    </button>
+                </Link>
+            );
+        }
+
+        return button;
+    }
+
+    render() {
+        const ariaLabel = `${this.props.name}, ${this.props.description}`.toLowerCase();
+        let versionLabel = `(${this.props.version})`;
+        if (this.props.installedVersion !== '') {
+            versionLabel = `(${this.props.installedVersion})`;
+        }
+
+        let pluginIcon;
+        if (this.props.iconData) {
+            pluginIcon = (
+                <div className='icon__plugin icon__plugin--background'>
+                    <img src={this.props.iconData}/>
+                </div>
+            );
+        } else {
+            pluginIcon = <PluginIcon className='icon__plugin icon__plugin--background'/>;
+        }
+
+        let localTag;
+        if (!this.props.downloadUrl) {
+            const localTooltip = (
+                <Tooltip id='plugin-marketplace__local-tooltop'>
+                    <FormattedMessage
+                        id='marketplace_modal.list.local.tooltip'
+                        defaultMessage='This plugin is not listed in the marketplace but was installed manually.'
+                    />
+                </Tooltip>
+            );
+
+            localTag = (
+                <OverlayTrigger
+                    delayShow={Constants.OVERLAY_TIME_DELAY}
+                    placement='top'
+                    overlay={localTooltip}
+                >
+                    <span className='tag'>
+                        <FormattedMessage
+                            id='marketplace_modal.list.local'
+                            defaultMessage='LOCAL'
+                        />
+                    </span>
+                </OverlayTrigger>
+            );
+        }
+
+        let pluginDetails = (
```

Comment:
I'm thinking this would be more readable, rather than using/reassigning the value.

```suggestion
        const pluginDetailsInner = (
```

Reply:
Good suggestion!